dev/kubernetes/k3d-dev-env-setup.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
```

Comment:
```suggestion
set -ex
```

Can we add `set -ex` to the shell scripts to make it easier to debug issues?

Reply:
I added `set  -x`

I modified this to first check if the registry or cluster already exist. The script was failing with `-e` due to the way I was checking if they already existed. Happy for suggestions on this file.